src/script/backport-create-issue

```diff
@@ -55,20 +57,9 @@ tracker2tracker_id = {}
 version2version_id = {}
 resolve_parent = None
 
-def usage():
-    logging.error("Command-line arguments must include either a Redmine key (--key) "
-                  "or a Redmine username and password (via --user and --password). "
-                  "Optionally, one or more issue numbers can be given via positional "
-                  "argument(s). In the absence of positional arguments, the script "
-                  "will loop through all issues in Pending Backport status.")
-    exit(-1)
-
 def parse_arguments():
     parser = argparse.ArgumentParser()
     parser.add_argument("issue_numbers", nargs='*', help="Issue number")
-    parser.add_argument("--key", help="Redmine user key")
-    parser.add_argument("--user", help="Redmine user")
-    parser.add_argument("--password", help="Redmine password")
```

Comment:
I'm fine to leave off the `--key` argument, but I don't understand why this PR is dropping `--user`/`--password`?

Reply:
My expectation would be that `--user`/`--password`, if given, would take precedence over a `~/.redmine_key` file.

Reply:
I had implemented that way before, but since we don't use it in `ceph-backport.sh` I assumed it was OK to remove it here.
I will revert the user/password removal.

Reply:
Done